# Fix for running on "next" branch (as built on 10/07/2024 DD/MM/YYYY)
diff --git a/examples/periodicPipe/pipe.udf b/examples/periodicPipe/pipe.udf
--- a/examples/periodicPipe/pipe.udf
+++ b/examples/periodicPipe/pipe.udf
@@ -1,7 +1,6 @@
 //
 // nekRS User Defined File
 //
-#include "udf.hpp"
 
 #ifdef __okl__
 
@@ -10,7 +9,7 @@
 
 #endif
 
-void UDF_ExecuteStep(nrs_t *nrs, dfloat time, int tstep)
+void UDF_ExecuteStep(double time, int tstep)
 {
   // print y1+ values at output steps
   if (nrs->isOutputStep)
@@ -28,6 +27,6 @@ void UDF_Setup0(MPI_Comm comm, setupAide &options)
 {
 }
 
-void UDF_Setup(nrs_t* nrs)
+void UDF_Setup()
 {
 }
